packages/terra-alert/docs/README.md

```diff
@@ -16,6 +16,12 @@ For more details, see [More Information][2].
 ```jsx
 import React from 'react';
 import Alert from 'terra-alert';
```

Comment:
Probably want to remove the now unnecessary call to Alert here.

Reply:
fixed here https://github.com/cerner/terra-core/pull/2407/commits/ffe3ff377f99567cc3e6d1d5c1be4eb714f01fe1.